cogs5e/dice.py

```diff
@@ -31,31 +31,59 @@ async def quick_roll(self, ctx, *, mod: str = '0'):
 
     @commands.command(name='roll', aliases=['r'])
     async def rollCmd(self, ctx, *, rollStr: str = '1d20'):
-        """Rolls dice in xdy format.
+        """Roll is used to roll any combination of dice in the `XdY` format. (`1d6`, `2d8`, etc)
+        
+        Multiple rolls can be added together as an equation. Standard Math operators and Parentheses can be used: `() + - / \*`
+
+        Roll also accepts `adv` and `dis` for Advantage and Disadvantage. Rolls can also be tagged with `[text]` for informational purposes. Any text after the roll will assign the name of the roll.
+
+        ___Examples___
+        `!r` or `!r 1d20` - Roll a single d20, just like at the table
+        `!r 1d20+4` - A skill check or attack roll
+        `!r 1d8+2+1d6` - Longbow damage with Hunter’s Mark
+
+        `!r 1d20+1 adv` - A skill check or attack roll with Advantage
+        `!r 1d20-3 dis` - A skill check or attack roll with Disadvantage
+
+        `!r (1d8+4)*2` - Warhammer damage against bludgeoning vulnerability
+
+        `!r 1d10[cold]+2d6[piercing] Ice Knife` - The Ice Knife Spell does cold and piercing damage
+
+        **Advanced Options**
+        __Operators__
+        Operators are always followed by a selector, and operate on the items in the set that match the selector.
+
+        Work on dice and numbers
```

Comment:
To be more consistent with the next subsection, which starts with `These operators only work on dice rolls.`, maybe this should be `These operations work on dice and sets of numbers.`

Reply:
Makes sense, updated the text and added in some information on sets